src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2741,6 +2775,311 @@ public void NullExceptionInLabeledStatement()
                 );
         }
 
+        [WorkItem(27866, "https://github.com/dotnet/roslyn/issues/27866")]
+        [Fact]
+        public void ParseElseWithoutPrecedingIfStatement()
+        {
+            UsingStatement("else {}",
+                // (1,1): error CS8641: 'else' cannot start a statement.
```

Comment:
Is the above ok? Instead of complaining about missing `if` we have an error related the `else` itself. Not sure if the previous version was actually better.

Reply:
I think the new error message is totally fine :)